components/library/gd2/patches/000-cmake-soname.patch

```diff
@@ -0,0 +1,170 @@
+--- libgd-gd-2.2.4/CMakeLists.txt.orig	2017-06-10 01:26:44.907495342 +0000
```

Comment:
Where is this patch coming from? (And how did they manage to decrease MAJOR version)?